faker/providers/ssn/el_GR/__init__.py

```diff
@@ -4,17 +4,31 @@
 
 class Provider(BaseProvider):
     """
-    A Faker provider for the Greek VAT IDs
+    A Faker provider for the Greek VAT IDs and Greek police IDs
```

Comment:
this is out of scope for this PR. Could  you submit it as separate PR?

Reply:
apologies, this was meant to be my next PR but somehow slipped in this one